tests/components/deconz/test_climate.py

```diff
@@ -112,8 +92,9 @@ async def test_climate_devices(hass):
     gateway.api.event_handler(state_changed_event)
     await hass.async_block_till_done()
 
-    thermostat = hass.states.get("climate.thermostat")
-    assert thermostat.state == "heat"
+    assert hass.states.get("climate.thermostat").state == "heat"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)